docs/30_data_standards.md

```diff
@@ -17,11 +17,12 @@ If you would like help determining which standard is most appropriate for your d
 NWB is designed to store a variety of neurophysiology data, including data from intracellular and extracellular electrophysiology experiments, data from optical physiology experiments, and tracking and stimulus data.
 The NWB team supports APIs in Python ([PyNWB](https://pynwb.readthedocs.io/)) and MATLAB ([MatNWB](https://github.com/NeurodataWithoutBorders/matnwb)), with tutorials for writing data broken down by experiment type.
 See the [NWB Tutorials](https://www.nwb.org/how-to-use/) page for more details.
-Also see the [NWB Conversion Tools user guide for converting data](https://nwb-conversion-tools.readthedocs.io/en/master/user_guide.html) for automated conversions from several popular proprietary data formats. 
+Also see the [NWB Conversion Tools user guide for converting data](https://nwb-conversion-tools.readthedocs.
+io/en/main/user_guide.html) for automated conversions from several popular proprietary data formats. 
```

Comment:
I'm not sure what happens if you break the hyperlink text in the middle, but unbreaking the line here keeps the change for this fix minimal, and it will allow people reading this as a text file to properly extract the link.

```suggestion
Also see the [NWB Conversion Tools user guide for converting data](https://nwb-conversion-tools.readthedocs.io/en/main/user_guide.html) for automated conversions from several popular proprietary data formats. 
```

Reply:
ok will remove the break